src/index.test.js

```diff
@@ -1,5 +1,3 @@
-/* eslint ava/no-only-test:0, ava/no-skip-test:0 */
-import test from 'ava'
```

Comment:
For this small project, moving from AVA to Jest was pretty simple because I actually was only using the AVA API in one place and all my tests are actually abstracted away from AVA. So I had to rewrite very little to migrate from AVA to Jest.